platform/jewel/ui/api/ui.api

```diff
@@ -1095,7 +1096,7 @@ public final class org/jetbrains/jewel/ui/component/ToggleableIconButtonState$Co
 }
 
 public final class org/jetbrains/jewel/ui/component/TooltipKt {
-	public static final fun Tooltip (Lkotlin/jvm/functions/Function2;Landroidx/compose/ui/Modifier;ZLorg/jetbrains/jewel/ui/component/styling/TooltipStyle;Landroidx/compose/foundation/TooltipPlacement;Lkotlin/jvm/functions/Function2;Landroidx/compose/runtime/Composer;II)V
+	public static final fun Tooltip-k8zXyTc (Lkotlin/jvm/functions/Function2;Landroidx/compose/ui/Modifier;ZLorg/jetbrains/jewel/ui/component/styling/TooltipStyle;Landroidx/compose/foundation/TooltipPlacement;ZJLkotlin/jvm/functions/Function2;Landroidx/compose/runtime/Composer;II)V
```

Comment:
Now that we are in IJP stable, we can't make binary-incompatible changes. You need to add new overloads whenever you add, change, or remove parameters, properties, etc.

This goes for both this and the changes to the styling.